spec/game-spec.js

```diff
@@ -0,0 +1,146 @@
+describe('Game', function () {
+  var game
+  var frame
+  var currentFrame
+
+  beforeEach(function () {
+    game = new Game()
+    game.startNextFrame()
+    currentFrame = game.getCurrentFrame()
+    frame = jasmine.createSpyObj('frame', ['getScore', 'addBowl', 'isFinished'])
```

Comment:
Oh I just love this setup. Each test is thoroughly independent, meaning you shouldn't ever leak state between your tests. Great great great